### ISSUE TITLE
Extra warning for "missing" status

### ISSUE THREAD

**Issue by nik123** [iterative/dvc#4729]
## Bug Report

`dvc status -c` reports about missing files twice:

```
$ dvc status -c foo.dvc
WARNING: Some of the cache files do not exist neither locally nor on remote. Missing cache files:
name: foo, md5: 0afbb537098c5a933ac62d91c52c1056.dir
        missing:            foo
```

### Please provide information about your setup

**Output of `dvc version`:**

```console
$ dvc version
DVC version: 1.8.4 (pip)
---------------------------------
Platform: Python 3.6.9 on Linux-4.15.0-117-generic-x86_64-with-Ubuntu-18.04-bionic
Supports: http, https, ssh
Cache types: hardlink, symlink
Repo: dvc, git
```

**Comment by nik123** [comment]
There was the same issue when I've tried fix #4383 so it's probably related: https://github.com/iterative/dvc/issues/4383#issuecomment-678809566

**Comment by nik123** [comment]
Cant' reproduce on another machine. Probably messed up my envirnoment.
